components/Features/constants.ts

```diff
@@ -0,0 +1,73 @@
+import { Icon, Linkedin, Slack } from 'react-feather';
+
+interface IIcon {
```

Comment:
These are not icons. You need to replace these. These are svg brand logos.[This is a design file](https://www.figma.com/file/O3IvMDpBVUBMzEU6ZuRdsX/Frontenship-Website?node-id=4%3A14026&t=fiILYcZsyTIptJtU-0)